aws-advanced-cicd/README.md

```diff
@@ -0,0 +1,96 @@
+# Advanced CI/CD for AWS using Pulumi and GitHub Actions
+
+> [!IMPORTANT]  
+> This is an advanced worksop that builds upon the [Getting started worksop](../aws-getting-started-cicd/). Please ensure its completion prior proceeding.
+
+In this workshop, you will learn advance topics that make up a robust infrastructure ci/cd pipeline through guided exercises. You will use Pulumi tooling to take your cloud infrastructure pipeline one step closer to production.
+
+This workshop introduces users to advance DevOps best practices. You will add compliance checks via policits, drift detection, and isolated test environments to an existing GitHub Actions pipeline. Accelerate your AWS projects with the skeleton code provided.
```

Comment:
```suggestion
This workshop introduces users to advanced DevOps best practices. You will add compliance checks via policies, drift detection, and isolated test environments to an existing GitHub Actions pipeline. This will help accelerate your AWS projects with the code examples provided.
```